src/app/administrator/administrator.component.html

```diff
@@ -11,35 +11,43 @@ <h5 class="card-title">Administrator</h5>
     />
     <div class="card-body">
       <h6 style="text-align: center;">
```

Comment:
There is another one of these but I need to put all the style rules in the .css file. I wrote these but I just wanted to note its bad practice lol.